scholia/api.py

```diff
@@ -313,8 +313,8 @@ def entity_to_full_text_url(entity):
     for statement in entity['claims'].get('P953', []):
         url = statement['mainsnak']['datavalue']['value']
         return url
-    else:
-        return ''
+
+    return ''
```

Comment:
The documentation to the function says that the function returns an empty string if the field is not set.